packages/apps/job-launcher/server/src/modules/job/job.dto.ts

```diff
@@ -112,7 +64,11 @@ export class JobImageLabelBinaryDto extends JobDto {
   @ApiProperty()
   @IsNumber()
   @IsPositive()
-  public requesterAccuracyTarget: number;
+  public minQuality: number;
+
+  @ApiProperty()
+  @IsString()
+  public gtUrl: string;
```

Comment:
We'll add here another field called `task_price` or `job_bounty` (discussing naming with cvat team) which will represent a reward for a user per job (10 pictures) at least for the first milestone.